backend/src/auth/strategy/user.strategy.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+
+import { JwtPayload } from '../../common/type/jwt-payload';
+import { JwtConfigService } from '../../config/auth/jwt/configuration.service';
+
+@Injectable()
+export class UserStrategy extends PassportStrategy(Strategy, 'user') {
+  constructor(private readonly jwtConfigService: JwtConfigService) {
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      secretOrKey: jwtConfigService.userSecretKey,
+    });
+  }
+
+  async validate(payload: JwtPayload) {
+    const token = {
+      userId: payload.userId,
+    };
+    return token;
```

Comment:
```suggestion
    return { userId: payload.userId };
```
제안드립니다. token 선언해서 명확한 게 좋다면 안하셔두됩니다잇